hypha/apply/funds/templates/funds/tables/table.html

```diff
@@ -52,7 +52,7 @@ <h6 class="heading heading--normal heading--no-margin">{% trans "Review outcomes
                                 {% for reviewer in submission.has_reviewed %}
                                     <li class="list__item list__item--has-reviewed ">
                                         <span class="list__item--reviewer-name">
```

Comment:
```suggestion
                                        <span class="list__item--reviewer-name truncate">
```

Use tailwindcss to make the intension clear with `truncate` class.